### ISSUE TITLE
Update to new govuk-frontend crown logo 

### ISSUE THREAD

**Issue by paulwaitehomeoffice** [govuk-react/govuk-react#1556]
govuk-frontend has updated the crown logo used in its header, following the coronation of King Charles III: https://github.com/alphagov/govuk-frontend/releases/tag/v5.1.0

I'd like govuk-react to update its crown logo to match govuk-frontend. This might involve updating https://github.com/penx/govuk-react-icon-crown — I'm not familiar with React, so I'm not sure what's involved.

The govuk-frontend crown logo now includes the "GOV.UK" text as part of the path within the SVG, whereas govuk-react crown logo seems to have the text in the HTML, so this might require a bit more rework than just swapping out an image file.

**Comment by penx** [comment]
@paulwaitehomeoffice yep that’s the repository that needs updating. I’d accept a PR to that repository that updates the SVG and bumps the major version number. 

Once that’s published, a PR to govuk-react will be needed to update the dependency here. 

I don’t work on govuk projects anymore and have little time to maintain these, but happy to assist anyone who is able to help with maintenance.

**Comment by paulwaitehomeoffice** [comment]
Gotcha, thanks @penx . So this repository is effectively unmaintained currently?

**Comment by penx** [comment]
I'm maintaining it in the sense of fixing any critical issues and overseeing community input.

I have not worked on govuk projects for several years and this project receives no funding other than a single github sponsor a last year.

If you use this project in a production govuk application then can I suggest you or your employer considers sponsoring it or assisting with maintenance?
